### ISSUE TITLE
PSR2.Methods.FunctionCallSignature.MultipleArguments error on short closure returning new object with multiple constructor arguments

### ISSUE THREAD

**Issue by sspat** [squizlabs/PHP_CodeSniffer#2572]
# Versions
codesniffer@dev-master
php 7.4.0beta1

# Description
When adding a short syntax closure as an argument in a multiline function call which returns a new object with multiple constructor arguments there is a false positive error.

```php
<?php

declare(strict_types=1);

throwException(
    fn () : Exception => new Exception('Message', 1)
);

function throwException(callable $function) : void
{
    throw $function();
}
```
```
6 | ERROR | [x] Only one argument is allowed per line in a multi-line function call (PSR2.Methods.FunctionCallSignature.MultipleArguments)
```
# Expected behavior
No errors should be raised by the PSR-2 rule, this for example passes the check:
```php
<?php

declare(strict_types=1);

throwException(
    fn () : Exception => new Exception('Message')
);

function throwException(callable $function) : void
{
    throw $function();
}
```

**Comment by gsherwood** [comment]
Closing as merged with #2523
